routes/loginRoutes.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const db = require('../data/helpers/helpers');
+const Joi = require('joi');
+const validate = require('../data/helpers/validations');
+
+const router = express.Router();
+const bcrypt = require('bcryptjs');
+
```

Comment:
Make sure to incorporate sessions in the server as that is the main focus of the assignment 👍